config/locales/fr.yml

```diff
@@ -106,6 +110,34 @@ fr:
       unsubscribe: Se désabonner
       username: Nom d’utilisateur⋅ice
       web: Web
+
+    account_moderation_notes:
+      account: Modérateur⋅ice 
+      created_at: Date
+      create: Créer
+      created_msg: Note de modération créée avec succès!
```

Comment:
En typographie française, il faut une espace avant un point d'exclamation.

Reply:
Thin non-breaking spaces « ! », cut the last one in case there is no text after « !».
The same with ? and :